portal/resources/views/admin/participant/list.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.admin')
+
+@section('content')
+<div
+    class="container"
+    id="participant-search"
+    data-url={{ route('participant.show', ['participant' => '_participant_'])  }}
+    data-participants='{!! json_encode($participants) !!}'>
```

Comment:
Don't know if it is in the used version of blade, but here we may use the `@json` helper:

```
<div data-participants="@json($participants)">
```